x-pack/plugins/cases/public/components/app/routes.tsx

```diff
@@ -70,7 +70,8 @@ const CasesRoutesComponent: React.FC<CasesRoutesProps> = ({
         </Route>
 
         <Route path={getCasesConfigurePath(basePath)}>
-          {permissions.all ? (
+          {/* TODO: should we add push here?*/}
```

Comment:
@cnasikas what do you think? Should the user be required to have `push` permissions to access the connector configuration page? They're really only making _updates_